### DISCUSSION TITLE
unocss 不支持小程序嘛？

### DISCUSSION THREAD

**Discussion by zeMinng** [unocss/unocss#3928] [Q&A]
unocss 不支持小程序嘛？
![image](https://github.com/unocss/unocss/assets/75962652/ae9b199e-fde1-4fff-ad9e-3900882f2090)

**Comment by xihuanxiaorang** [comment]
你可以试试安装这个预设：https://github.com/MellowCo/unocss-preset-weapp

**Reply by zeMinng** [reply]
thanks!

**Comment by elarbor** [comment]
我也试了，样式可以，图标不行

**Comment by xlzy905** [comment]
https://github.com/xlzy905/alipay-unocss-template

支付宝小程序template（微信也差不多）
